src/structures/Guild.js

```diff
@@ -432,9 +432,15 @@ class Guild extends Base {
   }
 
   /**
+   * A Discord Ban Object.
+   * @typedef {Object} Ban
```

Comment:
I'd call this `BanObject`, not just `Ban`, as it could be confused with other things. But thats just me

Reply:
i mean the description tells that tbh and fetchBans is the only method that is using that type so idk but you're maybe right